data/common.yaml

```diff
@@ -131,6 +131,7 @@ simp::server::data:
   - pam::wheel
   - selinux
   - svckill
+  - fips
```

Comment:
This is definitely related to https://github.com/simp/pupmod-simp-fips/pull/5, and not a typo.